packages/visualizations-react/stories/Chart/Formatting/LongDataLabels.stories.tsx

```diff
@@ -92,14 +92,16 @@ const pieDataFrame = [
     { x: 'Delta', y: 3000000 },
 ];
 
-export const PieLongDataLabels = storyWithArgs<Props<DataFrame, ChartOptions>>(Sample, {
+export const PieLongDataLabels = ChartTemplate.bind({});
+const PieLongDataLabelrsArgs: Props<DataFrame, ChartOptions> = {
     data: {
         loading: false,
         value: pieDataFrame,
     },
     options: {
         labelColumn: 'x',
         source: defaultSource,
+        // source: defaultSource,
```

Comment:
A small oversight to remove.

Reply:
Fixed here 8f65f9865ef34a950a8985836ed2c5300c5c32fd